ui/components/subscription/TeamJobModal.tsx

```diff
@@ -132,6 +150,24 @@ export default function TeamJobModal({
               ])
             }
 
+            //Get job id and team id from receipt and send discord notification
+            const jobId = parseInt(tx.receipt.logs[1].topics[1], 16).toString()
```

Comment:
FYI: i think something like this should work

const iface = new ethers.utils.Interface(JobsABI);
const parsedLog = iface.parseLog(tx.receipt.logs[1]);
const jobId = parsedLog.args.jobId
const jobTeamId = parsedLog.args.job